examples/hello-world/README.md

```diff
@@ -0,0 +1,39 @@
+# Hello World example
+Start building an app for Contentful with this hello-world boilerplate. It comes with the [App SDK](https://ctfl.io/app-sdk), our design system [Forma 36](https://ctfl.io/f36), and all the code you need to run a basic app. It will render a simple "Hello World!" in the [app configuration screen](https://www.contentful.com/developers/docs/extensibility/app-framework/locations/#app-configuration):
+
+![Screenshot](https://images.ctfassets.net/tz3n7fnw4ujc/2xWhCSYIJmGCJVVRHXw2Yb/0aa067e4f3d993e886031059cd5aabb6/hello-world-app.png)
+
+## How to use
+
+Install it and run:
+
+```bash
+npm install
+npm start
+# or
+yarn
+yarn start
+```
+
+To test, you can create an app definition in your Contentful organization settings pointing to `http://localhost:1234` and registering the `app-config` location.
+
```

Comment:
do we have a screenshot for that ? I think that could be helpful to have the basic setup displayed for an app definition with a location. Also we should inform the users about the `insecure content` issue when we try render to the app served via `http`. Maybe some users will think that is not expected.

Reply:
good point, added